README.md

```diff
@@ -26,10 +26,11 @@ $ kubectl directpv info
 
 4. Discover and add drives for volume scheduling.
 ```sh
-# Discover drives
-$ kubectl directpv discover --output-file drives.yaml
+# Discover drives to check the available devices in the cluster to initialize
+# The following command will create an init config file (default: drives.yaml) which will be used for initialization
+$ kubectl directpv discover
 
-# Review drives.yaml for drive selections and initialize those drives
+# Review the drives.yaml for drive selections and initialize those drives
```

Comment:
This formats the drives as XFS - should note this in case someone modifies the drives.yaml and includes drives that have existing data on them.

Reply:
yes, good point. we can actually remove the `fs` field from `drives.yaml` and probably add a comment saying that these drives will be formatted with XFS